libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -24,23 +24,26 @@ import {
     Component,
     ContentChild,
     EventEmitter,
+    forwardRef,
     Input,
     OnDestroy,
     OnInit,
     Optional,
     Output,
-    TemplateRef,
-    ViewEncapsulation,
     Provider,
-    forwardRef
+    TemplateRef,
+    ViewEncapsulation
 } from '@angular/core';
-import { ControlContainer, FormGroup, AbstractControl } from '@angular/forms';
+import { AbstractControl, ControlContainer, FormGroup } from '@angular/forms';
+import { KeyValue } from '@angular/common';
 import { Subject } from 'rxjs';
 import { coerceBooleanProperty } from '@angular/cdk/coercion';
 
 import { FormField } from '../form-field';
 import { FormGroupContainer } from '../form-group';
-import { LabelLayout, HintPlacement, FormZone } from '../form-options';
+import { HintPlacement, LabelLayout } from '../form-options';
+import { FormFieldGroup } from '../form-field-group';
+import { Field, FieldGroup, FieldColumn, isFieldChild, isFieldGroupChild, getField } from '../form-helpers';
 
 export const formGroupProvider: Provider = {
     provide: FormGroupContainer,
```

Comment:
Update all the absolete comments that describes and refers to older implementation with zLeft, zRight etc..
like below
```
 <fdp-form-field [id]="'Field A'" [rank]="10" zone="zLeft">
 ```